Added clients and wallet intro for libraries section (#646)
iota-community/iota-wiki: content/libraries/wallet.md

---
title: Wallet Library
description: 'Wallet libraries contain all the features you need to safely build wallets or integrations that require value-based
transfers, such as exchanges and pay-as-you-go systems.'
---

# Wallet Libraries

> Wallet libraries contain all the features you need to safely build wallets or integrations that require value-based

Reviewer: ```suggestion
Wallet libraries have everything that you need to safely build wallets or integrations that require value-based
```

Isn't it just one library?

transfers, such as exchanges and pay-as-you-go systems. 

![Layered Overview](/img/libraries/wallet_layers_overview.svg)

## wallet.rs

The [wallet.rs library](/wallet.rs/wallet.rs/welcome) is written
in [Rust](/wallet.rs/getting_started/rust), and
has bindings for [Java](/wallet.rs/getting_started/java)
[Java for Android](/wallet.rs/getting_started/java_for_android),
, [NodeJs](/wallet.rs/getting_started/nodejs)
, and [Python](/wallet.rs/getting_started/python).

Both the IOTA Foundation and the IOTA community maintain the library and this

> wiki, which means that you can rely on it as a source of knowledge on wallet.rs.

Reviewer: Same as above
